lib/iterable_flutter.dart

```diff
@@ -36,7 +36,13 @@ class IterableFlutter {
   }
 
   static Future<void> track(String event) async {
-    await _channel.invokeMethod('track', event);
+    await _channel.invokeMethod('track');
+  }
+
+  static Future<void> trackWithDataFields(
+      String event, Map<String, dynamic> dataFields) async {
+    await _channel.invokeMethod(
```

Comment:
Why do you want to have a new method that adds the data fields there?

To me, it should be the same method but with optional params to keep it in sync with the original sdk, WDYT?

Reply:
I added a new method last night cuz that was my current thought pattern. I then later had this similar thought that we could use the same method with optional params. I can go ahead and make that change.